packages/toolpad-app/src/utils/platform.ts

```diff
@@ -0,0 +1,4 @@
+export function isMac(): boolean {
```

Comment:
These kinds of utilities usually are pretty brittle and don't always work in my experience, do we still need it?

Reply:
Yeah, better to avoid exposing the platform, but abstract it instead. e.g.
```tsx
export function getModifierKey(): string {
  return isMac() ? '⌘' : 'Ctrl'
}
```

Nitpick though :)